tools/build.js

```diff
@@ -41,8 +41,10 @@ const getBuildOptions = input => {
         base = './js/masters/',
         debug = false,
         namespace = 'Highcharts',
+        product = 'Highcharts',
```

Comment:
What is the purpose (use case) of product in comparison to namespace?

Reply:
It's only used in replacing `@product.name@` one place in SVGrenderer. But I mostly wanted the build script to reflect more of the assembler options instead of setting them implicitly.